### ISSUE TITLE
Apple-clang 9.1 not supported

### ISSUE THREAD

**Issue by 013ez** [conan-io/conan#2700]
Please, add apple-clang 9.1 to supported platform. I had problem with compile missing libraries. I fixed it with manually edited ```.conan/settings.yml```

My CMake output:
```
-- The CXX compiler identification is AppleClang 9.1.0.9020039
```

Clang version: 
```sh
$ clang++ -v
Apple LLVM version 9.1.0 (clang-902.0.39.1)
Target: x86_64-apple-darwin17.5.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```

Conan version:
```
$ conan -v
Conan version 1.2.0
```

**Comment by lasote** [comment]
Thanks. We will add it to the next Conan version.

**Comment by memsharded** [comment]
Will be released in next 1.2.1

**Comment by lasote** [comment]
Released in 1.2.1
